Source/Csla.Blazor/ViewModel.cs

```diff
@@ -194,12 +332,31 @@ public T Model
       }
     }
 
-    private readonly Dictionary<string, object> _info = new Dictionary<string, object>();
+    /// <summary>
+    /// Gets or sets a value indicating whether the
+    /// view model should manage the lifetime of
+    /// the business object, including using n-level
+    /// undo.
+    /// </summary>
+    //[Browsable(false)]    // TODO: ask if this is desired
+    public bool ManageObjectLifetime { get; set; } = false;
+
+    /// <summary>
+    /// Gets a value indicating whether this object is
+    /// executing an asynchronous process.
+    /// </summary>
+    public bool IsBusy { get; protected set; } = false;
+
+    // TODO: should we implement IsDirty, IsValid, CanSave, CanCancel, CanCreate, CanDelete, CanFetch, CanRemove, CanAddNew, SetProperties()
```

Comment:
I'm not a fan of echoing things that can be bound directly off `vm.Model`. If implementing these things in the view model add functionality or enable scenarios that aren't otherwise available that's one thing, but if they just echo what's already available then that's extra code to maintain for no value.

Reply:
Agreed.